.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,51 @@
+name: E2E Tests
+on:
+  workflow_run:
+    workflows:
+      - "Vercel Deployment"
+      - "Vercel Deployment erd-sample"
+    types:
+      - completed
+
```

Comment:
This E2E test GitHub Action will start after the completion of the following workflows: ``Vercel Deployment erd-sample`` and ``Vercel Deployment``. The reason is that we need to access the deployed URL to perform the E2E tests.

This condition cannot be tested on this branch because it requires a merge to the main branch. However, I've confirmed that the test passes with a fixed URL.

with a fixed URL pattern: https://github.com/liam-hq/liam/blob/daeaf24255a64729b6a5d8520338e8e79f263282/.github/workflows/e2e-test.yml#L20

GitHub action log:
https://github.com/liam-hq/liam/actions/runs/13127470815/job/36626423092